README.md

```diff
@@ -1 +1,348 @@
-# dxFeed .NET API
+#  ![logo1][]![logo2][] dxFeed .NET API 
+
+
+[logo1]: https://github.com/Strotsev/dxfeed-net-api/blob/master/feed_logo.png
+[logo2]: https://github.com/Strotsev/dxfeed-net-api/blob/master/csharp_logo.png
+
+
+This package provides access to **[dxFeed](https://www.dxfeed.com/)** market data. The library is build as a thin wrapper over dxFeed **[C API library](https://github.com/dxFeed/dxfeed-c-api)**. 
+
+[![Release](https://img.shields.io/badge/release-8.3.0-blue)](https://github.com/dxFeed/dxfeed-net-api/releases/latest)
+[![License](https://img.shields.io/badge/license-MPL--2.0-orange)](https://github.com/dxFeed/dxfeed-net-api/blob/master/LICENSE)
+[![Downloads](https://img.shields.io/badge/downloads-347-green)](https://githubstats0.firebaseapp.com/)
```

Comment:
I think that it is necessary to create a separate ticket for setting up a dynamic check of the number of downloads. We already have a daily build check (in case Microsoft changes something without notice). You can add this functionality there.